test/libsolidity/smtCheckerTests/external_calls/call_with_value_2.sol

```diff
@@ -0,0 +1,15 @@
+contract C {
+	function g(address payable i) public {
+		require(address(this).balance == 100);
+		i.call{value: 0}("");
+		assert(address(this).balance == 100); // should hold
+		assert(address(this).balance == 20); // should fail
+	}
+}
+// ====
+// SMTEngine: all
+// ----
+// Warning 9302: (96-116): Return value of low-level calls not used.
+// Warning 6328: (120-156): CHC: Assertion violation might happen here.
```

Comment:
False positive? It's the line where the comment says "should hold".

Reply:
`might happen` means the solver couldn't solve it, so it's just an ack here